public/locales/de/common.json

```diff
@@ -100,7 +100,7 @@
   "selectLanguage": "Sprache auswählen",
   "trees": "Bäume",
   "logout": "Abmelden",
-  "directGiftRecipient": "Diese Spende unterstützt {{name}}",
+  "directGiftRecipient": "Diese Spende unterstützt {{name}}s Wald",
```

Comment:
This is often ok, e.g. `Norberts Wald`, but also sometimes wrong, e.g. `Klaus' Wald`.